jdk17/graphql/src/test/java/io/quarkus/ts/http/graphql/TracingIT.java

```diff
@@ -39,18 +38,23 @@ void verifyTracesInJaegerTest() {
         Assertions.assertEquals("Plato", classic.jsonPath().getString("data.friend.name"));
         Assertions.assertEquals("Plato", reactive.jsonPath().getString("data.friend_r.name"));
 
-        // the tracer inside the application doesn't send traces to the Jaeger server immediately,
-        // they are batched, so we need to wait a bit
-        await().atMost(60, TimeUnit.SECONDS).untilAsserted(() -> {
-            Response response = when()
-                    .get(jaeger.getTraceUrl() + "?service=" + SERVICE_NAME);
-            Assertions.assertEquals(HttpURLConnection.HTTP_OK, response.statusCode());
```

Comment:
Status code check is lost during the changes

Reply:
will add